pycontw/pycon.tw
# Add OTHER Location to event location and exclude them in the program …

…schedule

## Types of changes
Please put an `x` in the box that applies


## Description
**Describe what the change is**
Add a "Other" option to the Location field in Event-related models. And events with location set as Other are excluded to the program schedule.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to https://tw.pycon.org/2020/zh-hant/conference/schedule/
2. Select Day 2
3. You'll see Chia-liang Kao's talk at Goodideas studio.

## Expected behavior
It is arranged in Community Track but there is no way to specify the location of this SponsoredEvent.
After this change, we can choose Other in the location and it won't be included in the schedule (generator).

**Additional context**
This might not be an ideal solution since this is PyConTW 2020 only.
If we are going to have this kind of outing events in separate venues, maybe we should have another solution designed for this in the future.
